### PR TITLE
integrate download in sharing tests

### LINKED ISSUE #36075: review sharing tests in tests/acceptance/features/apiWebdavOperations
the tests that e.g. test the "download" in a share should be integrated into the sharing tests.

```
    When user "user0" shares folder "/FOLDER" with user "user1" using the sharing API
    Then the OCS status code should be "<ocs_status_code>"
    And the HTTP status code should be "200"
    And the fields of the last response should include
    .....
    And  the downloaded content when downloading file "/FOLDER/child.txt" for user "user1" with range "bytes=1-7" should be "wnCloud"
   .....
```

the action that we want to test is the "sharing" and that should result in a list of `Thens` and one of them is that the user can download the file

CC @phil-davis

### PR DESCRIPTION
**Pull request by haribhandari07**
## Description

This PR integrates downloaded and content checking operation in the previously created share scenarios.
 This PR also removes ```apiShareOperations/downloadFromShare.feature``` feature file and some scenarios from ```apiWebdavOperations/downloadFile.feature ``` as download is integrated into the previous share steps.
## Related Issue

- Fixes https://github.com/owncloud/core/issues/36075
## How Has This Been Tested?

CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Review by dpakach — COMMENTED** [review]
LGTM :+1:

**Comment by individual-it** [comment]
there should be now some scenarios that are obsolete and can be deleted, where are those?

**Comment by HariBhandari07** [comment]
@individual-it I have removed the obsolete sharing tests. There are many tests but with permissions attached, so I didn't remove them.

File: tests/acceptance/features/apiShareOperations/downloadFromShare.feature [hunk 1]
```diff
@@ -46,31 +46,6 @@ Feature: sharing
...
-
-  Scenario: Download a file that is in a folder contained in a folder that has been shared with a group with default permissions
-    Given user "user1" has been created with default attributes and skeleton files
-    And group "grp1" has been created
-    And user "user1" has been added to group "grp1"
-    When user "user0" has shared folder "PARENT" with group "grp1"
-    Then the downloaded content when downloading file "/PARENT (2)/CHILD/child.txt" for user "user1" with range "bytes=1-7" should be "wnCloud"
-
-  @smokeTest @public_link_share-feature-required
-  Scenario: Download a file that is in a folder contained in a folder that has been shared with public with default permissions
```
Comment (individual-it) [inline tests/acceptance/features/apiShareOperations/downloadFromShare.feature]: cannot see that test being added somewhere else

File: tests/acceptance/features/apiShareOperations/downloadFromShare.feature [hunk 2]
```diff
@@ -119,14 +94,4 @@ Feature: sharing
       | shareType   | group  |
       | shareWith   | grp1   |
       | permissions | read   |
-    Then the downloaded content when downloading file "/PARENT (2)/CHILD/child.txt" for user "user1" with range "bytes=1-7" should be "wnCloud"
-
-  @public_link_share-feature-required
-  Scenario: Download a file that is in a folder contained in a folder that has been shared with public with Read only permission
```
Comment (individual-it) [inline tests/acceptance/features/apiShareOperations/downloadFromShare.feature]: cannot see this test being added somewhere

**Review by individual-it — CHANGES_REQUESTED** [review]

**Review by individual-it — APPROVED** [review]

**Pull request merged**